web/modules/custom/joinup_licence/joinup_licence.post_update.php

```diff
@@ -42,3 +42,10 @@ function joinup_licence_post_update_import_legal_type_vocabulary() {
 function joinup_licence_post_update_eupl() {
   \Drupal::service('module_installer')->install(['eupl']);
 }
+
+/**
+ * Re import the legal type vocabulary so that the weight is imported.
+ */
+function joinup_licence_post_update_re_import_legal_type_vocabulary() {
+  joinup_licence_post_update_import_legal_type_vocabulary();
```

Comment:
If this will happen couple of times in the future, why not moving the logic into a service. Just calling a previous post-update looks odd.

Reply:
I have provided a service for this case. It seems a bit overkill for this case but this service can easily be extended for every fixture that we have (a simple array entry).